types/global.d.ts

```diff
@@ -1,6 +1,11 @@
 export {};
 
-type Range = [string, number, number]; // style, location, length
+type Range = {
+  style: string;
+  start: number;
+  length: number;
+  depth?: number;
+}; // style, location, length
```

Comment:
Let's remove this comment since it's no longer relevant
```suggestion
};
```